assets/js/eventCardFetcher.js

```diff
@@ -23,36 +23,30 @@ $(function () {
     function renderCard(post) {
         var splitTitle = post.title.split(":"); // <!> This assumes all event titles contain a colon
         var splitDate = post.custom_excerpt.split("-");
+        // prettier-ignore
         var html =
             '<article class="border-2 border-gray-400 rounded-xl p-2.5 event-card">\
-                        <div class="grid grid-cols-2">\
-                            <div>\
-                                <p class="text-yellow-500 font-semibold text-3xl event-date-day">' +
-            splitDate[2] +
-            '</p>\
-                                <p class="text-yellow-500 text-xl event-date-month">' +
-            monthNames[parseInt(splitDate[1]) - 1] +
-            '</p>\
-                            </div>\
-                            <div>\
-                                <p class="text-3xl text-gray-500 text-right">➔</p>\
-                            </div>\
-                        </div>\
-                        \
-                        <div class="my-2.5">\
-                            <p class="text-gray-600 font-semibold text-3xl py-0.5">' +
-            splitTitle[0] +
-            ':</p>\
-                            <p class="text-gray-600 font-semibold text-xl">' +
-            splitTitle[1] +
-            "</p>\
-                        </div>\
-                    </article>";
+                <div class="grid grid-cols-2">\
+                    <div>\
+                        <p class="text-yellow-500 font-semibold text-3xl event-date-day">' + splitDate[2] + '</p>\
```

Comment:
Could this line be modified to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals? Same goes for L32, 40, 41.